src/omniperf_base.py

```diff
@@ -55,6 +55,11 @@
     "gfx942": {"mi300": ["MI300A_A1", "MI300X_A1"]},
 }
 
+MI300_CHIP_IDS = {
+    "29856" : "MI300A_A1",
+    "29858" : "MI308X",
+}
+
```

Comment:
The minimum requirements for the chip id map is MI300X_A1, MI300A_A1, and MI308X. These are required because they're both indistinguishable as gfx942 arch. The others are nice to have.

Please update with MI300A_A1.

Reply:
Hi Cole, I've added `MI300X_A1` and updated `populate_mspec()` in the latest commit. Thank you!